addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6818,7 +6818,18 @@ msgctxt "#13516"
 msgid "Add art"
 msgstr ""
 
-#empty strings from id 13517 to 13549
+#: system/settings/settings.xml
+msgctxt "#13517"
+msgid "Threshold when atempo resampling kicks in"
```

Comment:
This is not a proposed help text but some background info:

For minor speed adjustments resampling is the preferred method. In regard to a/v sync it is more accurate. I am not completely sure but I think audio quality is also better.
For larger speed adjustments resampling is not ideal because it changes the pitch -> chipmunk voices. With v17 we introduced ffmpeg's atempo filter but we don't have too much experience with it yet. The atempo filter tries to keep the audio pitch.